bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson_WkkWr/Triboson_M_R/Triboson_M_R_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card kkinputs 5 <MASS>
+set param_card kkinputs 2 6.708000e+00
+set param_card mass 9000025 <MASS_R>
+set param_card mass 9000024 <MASS>
+set param_card DECAY 9000024 1.195600e+01
+set param_card DECAY 9000025 0.31035
```

Comment:
Small comment on these settings? Where are these values coming from? As it seems not just to be a small fixed number, maybe the same logic can be applied to all mass points?